src/extensions/default/CodeSnippetsAdder/main.js

```diff
@@ -0,0 +1,34 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager = brackets.getModule("command/CommandManager");
+	var EditorManager  = brackets.getModule("editor/EditorManager");
+
+	var CMD_ADD_CODE_SNIPPET_TEXT   = "Add Code Snippets";
+    var CMD_ADD_CODE_SNIPPET_ID     = "bramble.addCodeSnippet";
+    
+	function addCodeSnippet(snippet) {
+		var editor = EditorManager.getCurrentFullEditor();
+		if (editor) {
+			var insertionPos = editor.getCursorPos();
```

Comment:
What happens here if the cursor isn't in the current editor?  What are the possible return values of `getCursorPos()`?  Will it do the right thing in all cases?